configs/luslab-base.config

```diff
@@ -75,27 +75,11 @@ dag {
   file = "${params.tracedir}/pipeline_dag.svg"
 }
 
-// Create process scaling defaults for running in various envoironments
-/*
-Selectors priority
-When mixing generic process configuration and selectors the following priority rules are applied (from lower to higher):
-
-1. Process generic configuration.
-2. Process specific directive defined in the workflow script.
-3. withLabel selector definition.
-4. withName selector definition.
-*/
+// Default run params
 process {
-//accelerator
-
-  // Memory and time increases with the number of retrys
-  cpus = { check_max( 2, 'cpus' ) }
-  memory = { check_max( 4.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
```

Comment:
how come task.attempt was taken out here?

Reply:
Becuase its now in the crick config. It doesn't make sense as a default as it only applies to clusters